crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -462,7 +485,9 @@ mod tests {
             },
             &domain_separator,
             settlement_contract,
+            &MockAuthenticator {},
```

Comment:
nit: I'm actually a bit surprised that clippy doesn't complain about that.
```suggestion
            &MockAuthenticator,
```